tests/helpers/linux/__init__.py

```diff
@@ -12,9 +12,10 @@
 import unittest
 
 import drgn
+from tests import TestCase
 
 
-class LinuxHelperTestCase(unittest.TestCase):
+class LinuxHelperTestCase(TestCase):
```

Comment:
A less intrusive way to do this would be to isolate all of the vmcore tests in their own directory. When running the normal tests, we could still skip the vmcore tests due to `/proc/vmcore` not existing, but then the test command could explicitly run only that directory when running in kdump. What do you think, is that better or does separating the vmcore tests have any downsides?

Reply:
That's definitely a better approach, I'll migrate to that.

Reply:
Do you want me to revert the rest of the tests that I changed to inherit from `TestCase` instead of `unittest.TestCase`, or should I keep that change (the majority of the tests already inherited from `TestCase`, so it makes sense to make things uniform IMO).

Reply:
It's up to you, but if you keep the change, make it a separate commit with its own justification.